nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3DirectBatchHandler.java

```diff
@@ -101,7 +98,16 @@
 
     public static final String INFO_USE_S3_ACCELERATE = "useS3Accelerate";
 
-    public static final String INFO_POLICY_TEMPLATE = "policyTemplate";
+    protected static final String AWS_STS_POLICY = "{\n" +
```

Comment:
I am not keen on having this hardcoded.  It doesn't look like it changes much but is is possible to have it via a config file?

Reply:
Yes good point. Let's have a configuration property to allow overriding this.